views/add_event.hbs

```diff
@@ -1,79 +1,143 @@
-<div class="add-eventContainer">
-  <h3 class="userAddEvent">Add Event</h3>
-
+{{>header}}
   <form class="add-event-form" action="/add-event" method="post">
     <label>Name </label>
-    <input class="form-control" type="text" name="name_english" placeholder="English" required>
-    <input class="form-control" type="text" name="name_arabic" placeholder="Arabic">
+    <input class="form-input" type="text" name="name_english" placeholder="English" required>
```

Comment:
Not really any need for this `form-input` class, easier to just use `add-event-form input` as the selector, and have cleaner html - same with all the labels (atm you have 4 separate label classes, when there are only 2 different styles of labels on the page)